src/main/scala/edu/ie3/util/scala/quantities/QuantityConversionUtils.scala

```diff
@@ -243,4 +242,27 @@ object QuantityConversionUtils {
     )
   }
 
+  /** Implicit class that contains a method to convert a given
+    * [[ComparableQuantity]] with unit [[CELSIUS]] or [[KELVIN]] into
+    * [[squants.thermal.Celsius]] or [[squants.Kelvin]].
+    *
+    * @param quantity
+    *   To convert.
+    */
+  implicit class TemperatureConversionSimona(
+      quantity: ComparableQuantity[Temperature]
+  ) {
+
+    /** Returns a quantity with unit [[Celsius]].
+      */
+    def toSquantsCelsius: squants.Temperature = Celsius(
+      quantity.to(CELSIUS).getValue.doubleValue
+    )
+
+    /** Returns a quantity with unit [[Kelvin]].
+      */
+    def toSquantsKelvin: squants.Temperature = Kelvin(
+      quantity.to(KELVIN).getValue.doubleValue
+    )
```

Comment:
Do we need the second method? If not, it can be removed and the first method could be renamed to `toSquants` like most of the conversion methods.

Reply:
I remember that it was sometimes, no idea why, important to convert to Celsius or Kelvin. Therefor I guess we need both.

Reply:
But we are using squants. We can convert the quantity to Kelvin, if we need it at some point.

Reply:
I see the danger, that someone will use this directly on Kelvin which the get's converted into Celsius. But let's see...